TrainingManagementSystem/src/main/java/trainingManagementSystem/controller/UserController.java

```diff
@@ -0,0 +1,37 @@
+package trainingManagementSystem.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.ModelMap;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+import trainingManagementSystem.model.User;
+import trainingManagementSystem.service.UserServices;
+
+@Controller
+@RequestMapping
```

Comment:
```suggestion
@RequestMapping("users")
```
mặc định action vào controller này sẽ có kiểu bắt đầu là `users` nhé